metrics/web/lib/common/presentation/metrics_theme/config/dimensions_config.dart

```diff
@@ -13,11 +13,11 @@ class DimensionsConfig {
   static const double buttonHeight = 48.0;
 
   /// An outer diameter of the graph indicator.
-  static const double graphIndicatorOuterDiameter = 5.33;
+  static const double graphIndicatorOuterDiameter = 7.0;
```

Comment:
It seems like the indicator is `10.0` in diameter now: 
![image](https://user-images.githubusercontent.com/70939496/108198405-3fea1980-7124-11eb-90b8-b9e273c416fa.png)